community/logo_request.md

```diff
@@ -0,0 +1,9 @@
+## Member Company logos
+
+Member Companies as defined [here](readme.md#community-roles) can request their logo be displayed on https://onnx.ai and other materials.
+
+To have your logo displayed, submit a PR to https://github.com/onnx/onnx.github.io with the following:
+1. Text of the PR must include written permission indicating the logo can be used on the onnx.ai website as well as in presentations showing ONNX Member Companies
+2. A high quality logo file with transparent background needs to be committed in the "logos" directory
```

Comment:
Is there any guidance from the design team for a specific requirement on the definition of "high quality" (e.g. file size, pixel dimensions, etc)? Otherwise LGTM.

Reply:
@jspisak do you have a recommendation?

Reply:
@prasanthpul - we typically request 300dpi (best if physical printing is required or a vector file which can be saved in any resolution. I would add this language to the requirements.

Reply:
updated